pkg/pipelineascode/cancel_pipelineruns.go

```diff
@@ -24,18 +24,21 @@ var cancelMergePatch = map[string]interface{}{
 }
 
 func (p *PacRun) cancelPipelineRuns(ctx context.Context, repo *v1alpha1.Repository) error {
-	if p.event.TriggerTarget != "pull_request" {
```

Comment:
That condition is not there anymore, maybe stil check if nothing is pull_request or push?

Reply:
I see condition is there in current main branch
https://github.com/openshift-pipelines/pipelines-as-code/blob/main/pkg/pipelineascode/cancel_pipelineruns.go#L27C2-L31C3

Reply:
that's not what i meant, i mean if p.event.TriggerTarget != "pull_request" ||  p.event.TriggerTarget != "push"